phobos/simple_deployment.md

```diff
@@ -0,0 +1,182 @@
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor01-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor02-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor03-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor04-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor05-c_
+```
+
+
+## Storage Nodes favor ironic-rpc-storage-interim additional drives
+/dev/sdb | 3TB
+...
+/dev/sdk | 3TB
+
+
+## Phobos Xenial image is set to not allow root user to log in
+To remove the restriction run this on each osd node and on rsyslog
+```bash
+ssh ubuntu@<ip>  "sudo sed -i -e 's/^no-port-forwardin.*\(ssh-rsa.*$\)/\1/g' /root/.ssh/authorized_keys"
+```
+
+
+##  hosts file
+Create entries in /etc/hosts for all servers, specifying the IP address to which we can reach the servers via ssh:
+Example **/etc/hosts** snippet:
+```bash
+/etc/hosts
+172.20.41.X    ceph-admin01-c_ 
+172.20.41.X    rsyslog01-c_
+172.20.41.X    ceph-stor01-c_
+172.20.41.X    ceph-stor02-c_
+172.20.41.X    ceph-stor03-c_
+172.20.41.X    ceph-stor04-c_
+172.20.41.X    ceph-stor05-c_
+```
+
+## Passwordless root ssh 
+Configure password-less root ssh login from the deployment host to all other hosts. Methods will vary, but typically involve generating a ssh-keypair on the deployment host and using ssh-copyid to distribute the public key to each other node.
+
+On the admin vm create an **ssh-key**
+```bash
+ssh-keygen -t ed25519 -C cephdeploy
+```
+
+Push the public key generated to each server in the cluster
+```bash
+ ssh-copy-id -i ~/.ssh/id_ed25519 root@<server ip>
+```
+
+
+## Clone rpc-ceph repo and set up directories
+Clone repo
+```bash
+cd /opt && git clone https://github.com/rcbops/rpc-ceph.git
+```
+Set up phobos directory
+```bash
+cd rpc-ceph && mkdir phobos
+```
+
+
+## Install and Setup up Ansible
+
+```bash
+apt-get update
+apt install --yes python-pip
+./scripts/bootstrap-ansible.sh 
+```
+
+# Setup Inventory File
+Create file /opt/rpc-ceph/phobos/inventory
+```
+[ceph-cluster]
```

Comment:
I'm not sure what you use the ceph-cluster group for, you could change this to [all]
If you specify the ansible_host="172...." bit after each hosts's entry in [all] you can avoid having to respecify the ansible_host setting each time the host comes up.